x-pack/plugins/security_solution/public/flyout/left/hooks/use_fetch_alerts.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+import { useQuery } from '@tanstack/react-query';
+import type { AggregationsAggregate, SearchResponse } from '@elastic/elasticsearch/lib/api/types';
+import { isNumber } from 'lodash';
+import { useKibana } from '../../../common/lib/kibana';
+import { type AlertsQueryParams, createFindAlerts } from '../services/find_alerts';
+
+export type UseAlertsQueryParams = AlertsQueryParams;
+
+export interface UseAlertsQueryResult {
+  /**
+   * Was there an console.error
```

Comment:
nit: I don't understand this comment.

Reply:
console.error was an autocompletion probably :D thanks for the catch